lib/helper/FileSystem.js

```diff
@@ -55,6 +55,14 @@ class FileSystem extends Helper {
     assert.ok(fileExists(this.file), `File ${name} not found in ${this.dir}`);
   }
 
+  /**
+  * Checks that any file exists including given text
+  */
+  seeFileNameMatching(text) {
+    assert.ok(this.getFileNames().some(file => file.includes(text)),
+      `File contains ${text} not found in ${this.dir}`);
```

Comment:
maybe `File name which contains ${text} not found in ${this.dir}`);` to be clearer?